src/components/calcite-tip-manager/calcite-tip-manager.tsx

```diff
@@ -192,6 +192,7 @@ export class CalciteTipManager {
       // TODO: Empty state
       return <Host />;
     }
+    const environmentDir = this.el.closest("[dir=rtl]") ? "rtl" : "ltr";
```

Comment:
Can you use the `getElementDir` util instead? Also, for consistency, rename to `dir`.

Reply:
in other components we just call the const `rtl`